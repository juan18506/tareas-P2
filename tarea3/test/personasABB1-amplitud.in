crearArbolPersonas

# Construyo arbol 
crearPersona 7 7 Carlos
insertarPersonaArbolPersonas

crearPersona 20 20 Caroline
insertarPersonaArbolPersonas

crearPersona 2 2 Romina
insertarPersonaArbolPersonas

crearPersona 1 1 Jose
insertarPersonaArbolPersonas

crearPersona 13 Pedro
insertarPersonaArbolPersonas

crearPersona 8 8 Ana
insertarPersonaArbolPersonas

crearPersona 6 6 
insertarPersonaArbolPersonas

# La maxima amplitud es 3
amplitudArbolPersonas

liberarArbolPersonas

Fin