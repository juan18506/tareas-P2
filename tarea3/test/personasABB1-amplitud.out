1>2>#  Construyo arbol .
3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>#  La maxima amplitud es 3.
18>La amplitud del Ã¡rbol es: 3.
19>20>Fin.
